PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/helpers/UIHelperImpl.java

```diff
@@ -289,6 +294,28 @@ public void run() {
     }
     
     @Override
+    public void openEditor(@NotNull Node node) {
```

Comment:
Add a method for openredisexplorer. And make openEditor as a util method and private